### PR TITLE
chore(lint): adds 'no-eslint-disable' rule

### PR DESCRIPTION
**Pull request by alber70g**

### CODE REVIEW

File: packages/apps/explorer/.eslintignore [hunk 1]
```diff
@@ -3,3 +3,5 @@
 !.*.js
 **/dist/**
 **/coverage/**
+**/*.ts
```
Comment (jermaine150) [inline packages/apps/explorer/.eslintignore:6]: What's the reason these files are ignored / is there a plan to revisit them later?

Comment (alber70g) [reply packages/apps/explorer/.eslintignore:6]: We have a bunch of React specific linting errors/warnings that I'm not sure how to deal with yet. Especially every function needs to have a return type, which basically for every component is `JSX.Element`. Maybe it'd be better to customize a react set of rules for some projects

Comment (jermaine150) [reply packages/apps/explorer/.eslintignore:6]: Got it. As discussed in Slack we'll take it offline and discuss on how to handle these.

**Review by jermaine150 — DISMISSED** [review]

**Review by nillo — DISMISSED** [review]
besides ts comments lgtm

**Review by Randynamic — APPROVED** [review]

**Review by jermaine150 — APPROVED** [review]

**Pull request merged**